bin/gen-make-dep-chain

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+(for p ; do
```

Comment:
This script is not used right now, committed as I thought I would use it, `tac` is cat that reverses lines, `for p;` is equivalent to `for p in "@";`